### PR TITLE
Adding Resonance Technician, minor fixes 

### PR DESCRIPTION
**Pull request by dracontes**
**Tested card script**
- Resonance Technician

**Minor fixes**
- Mostly cleaning up cosmetic typos

Hopefully I've rebased my fork properly as was recommended elsewhere. Nice to see dragging and dropping folders works when  uploading files too.

### CODE REVIEW

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/b/brotherhoods_end.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ Types:Sorcery
 A:SP$ Charm | Choices$ DBDamage,DBDestroy | CharmNum$ 1
 SVar:DBDamage:DB$ DamageAll | ValidCards$ Creature,Planeswalker | NumDmg$ 3 | StackDescription$ SpellDescription | SpellDescription$ CARDNAME deals 3 damage to each creature and each planeswalker.
 SVar:DBDestroy:DB$ DestroyAll | ValidCards$ Artifact.cmcLE3 | SpellDescription$ Destroy all artifacts with mana value 3 or less.
-Oracle:Choose one -\n•Brotherhood's End deals 3 damage to each creature and each planeswalker.\n•Destroy all artifacts with mana value 3 or less.
+Oracle:Choose one -\n•  Brotherhood's End deals 3 damage to each creature and each planeswalker.\n•Destroy all artifacts with mana value 3 or less.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/b/brotherhoods_end.txt:7]: @tool4ever is that change right?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/b/brotherhoods_end.txt:7]: oh yea that seems to be one space too much / not at the other mode
I'll get it changed

Comment (Hanmac) [reply forge-gui/res/cardsfolder/b/brotherhoods_end.txt:7]: I mean do we need a space after the dot?
The second one doesn't have it either

Comment (tool4ever) [reply forge-gui/res/cardsfolder/b/brotherhoods_end.txt:7]: that seems to be how Oracle does it
e. g. _Rakdos Charm_

Comment (dracontes) [reply forge-gui/res/cardsfolder/b/brotherhoods_end.txt:7]: Well, I came to that change through finding, while searching the scripts, that it's far more common for the "newline escape+bullet character" combo to have whitespace after it (1274 hits for it against 21 hits for the "no whitespace" option). 
And, FWIW, that's what happens on Scryfall too (see [Brotherhood's End](https://scryfall.com/card/bro/128/brotherhoods-end)).
image

Comment (dracontes) [reply forge-gui/res/cardsfolder/b/brotherhoods_end.txt:7]: Oops. Didn't notice the double space on one and that I missed the second `\n•` instance. I'll be more vigilant about that in the future then.

**Pull request merged**